pupil_src/shared_modules/player_methods.py

```diff
@@ -58,7 +58,7 @@ def correlate_data(data, timestamps):
             # we might loose a data point at the end but we dont care
             break
 
-        if datum['timestamp'] <= ts:
+        if datum.get('timestamp', 0.) <= ts:
```

Comment:
why make timestamp optional? I think we should require this. Data without timestamp can simply not be correlated...